04.bowling/rubybowling.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << s
+end
```

Comment:
```suggestion
frames = shots.each_slice(2).to_a
```

[nits] こんな感じでも同じ結果になりますよ